deploy/cloudformation/manifest-pipeline-pipeline.yml

```diff
@@ -104,17 +112,18 @@ Resources:
       Path: /
   CloudFormationRolePolicy:
     Type: 'AWS::IAM::Policy'
-    DependsOn: CloudFormationTrustRole
-    Description: Setting IAM policy for the service role for AWS CloudFormation
     Properties:
       PolicyName: CloudFormationRolePolicy
       PolicyDocument:
         Statement:
+          # Allow the manifest-pipeline.yml to create any state machines it needs, using its stack name as a base for the name
           - Action:
-              - 'States:CreateStateMachine'
-              - 'States:DeleteStateMachine'
-            Resource: '*'
+              - 'states:CreateStateMachine'
+              - 'states:DeleteStateMachine'
+            Resource:
+              - !Sub 'arn:aws:states:${AWS::Region}:${AWS::AccountId}:stateMachine:*'
```

Comment:
Should this look like 
` -!Sub 'arn:aws:states:${AWS::Region}:${AWS::AccountId}:stateMachine:${AWS::StackName}*'` instead?

Reply:
IIRC this one only allowed for *. Can triple check

Reply:
I couldn't find anything in the docs either way. Resolving